nixos/modules/services/system/dbus.nix

```diff
@@ -65,31 +63,13 @@ in
         '';
         default = "disabled";
       };
-
-      socketActivated = mkOption {
```

Comment:
Shouldn't this be declared as a removed module?

Reply:
It was declared as removed for the past two years. This pr removes that marking.

Reply:
IIRC we have some specific list or option for removed options; this declaration was home-brewed.

Reply:
Yes, that was used in the past: https://github.com/NixOS/nixpkgs/commit/f292a27f442d10de4827800d064a3a8c64d05cee
Then it was switched to the custom one: https://github.com/NixOS/nixpkgs/commit/94819fdb5f0b624fd4e8233a064c0eaa1479c170

Either way, the removal has been notified for a sufficient time now.